lms/static/js/student_account/views/LoginView.js

```diff
@@ -86,6 +92,16 @@
 
             resetEmail: function() {
                 this.element.hide( this.$errors );
+                this.resetMessage = this.$resetSuccess.find('.message-copy');
+                if (this.resetMessage.find('p').length == 0){
```

Comment:
You have a diff quality failure by using ``==`` instead of ``===``. Also, the new linter will complain about the lack of a space between the close paren and the opening bracket.